src/server/api/controllers/port.controller.js

```diff
@@ -37,19 +37,15 @@ const createPort = async ({ body }) => {
       .insert({
         name: name
       })
-      .then(portID => {
-        if (waypoints.length !== 0) {
-          return knex("waypoints").insert({
-            suggested_route_id: suggested_route_id || null,
-            longitude: waypoints[0].longitude,
-            latitude: waypoints[0].latitude
-          });
-        } else {
-          return `No Waypoint Added for port: '${name}' `;
-        }
-      })
-      .catch(error => {
-        return `Error occured!  ${error.message}`;
+      .then(function([id]) {
+        const portWaypoint = waypoints.map(waypoint => {
+          return {
+            port_id: id,
+            longitude: waypoint.longitude,
+            latitude: waypoint.latitude
+          };
+        });
```

Comment:
Actually, this was more correct before. I want you to assume that this request body looks like this (unlike what I wrote in Trello):
```
{
    "name": "Port sunshine",
    "sailtime": 22.3,
    "waypoint":
    {
        "lon": 55.65646,
        "lat": 12.53635
    }
}
```
Notice that waypoints is no longer inside an array. This makes it more explicit that creating a port only takes one waypoint.

Reply:
Changes have been incorporated as requested. Also updated Swagger.json to reflect the changes.